cli/cmd/service_instances.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/cncf/servicebroker/k8s/service_controller/model"
+	"github.com/spf13/cobra"
+)
+
+const (
+	SERVICE_INSTANCES_URL = "/v2/service_instances"
+)
+
+var (
+	service         string
+	plan            string
+	servicePlanGUID string
+	parameters      string
+)
+
+func init() {
+	RootCmd.AddCommand(serviceInstancesCmd)
+	serviceInstancesCmd.AddCommand(createServiceInstancesCmd)
+	createServiceInstancesCmd.Flags().StringVarP(&name, "name", "n", "", "Name to give to Service Instance")
+	createServiceInstancesCmd.Flags().StringVarP(&service, "service", "s", "", "Name of the service to instantiate")
+	createServiceInstancesCmd.Flags().StringVarP(&plan, "plan", "p", "", "Name of the service plan to instantiate")
+	createServiceInstancesCmd.Flags().StringVarP(&spaceGUID, "space_guid", "g", "default", "Space GUID on which to instantiate the service to")
+	createServiceInstancesCmd.Flags().StringVarP(&parameters, "parameters", "a", "", "Parameters to pass to the service broker for creation (must be JSON object)")
+
+}
+
+var serviceInstancesCmd = &cobra.Command{
+	Use:   "service-instances",
+	Short: "Manage service instances",
+	Long:  "Manage service instances",
+}
+
+var createServiceInstancesCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create a service instance",
+	Long:  "Create a service instance",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(name) == 0 {
```

Comment:
fine, but no space-guid, it should be a flag like in the other cases. Also, let's figure out what this even means, it's a CFism, and seems like it should not be a required argument as per other comments.